dts/bindings/mmc/mmc.yaml

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2019, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: MMC/SDHC module
+version: 0.1
+
+description: >
+    This binding specifies the MMC/SDHC module.
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      generation: define
+      constraint: "zephyr,mmc-sdhc"
```

Comment:
We should not use a constraint in the generic mmc.yaml. This should defined in devices specific yaml file (e.g. nxp,imx-usdhc.yaml in your case). You should also take into account PR #16693 which adds a common base.yaml file.

Reply:
I agree. Please check the latest code pushed.